src/textual/widgets/_list_view.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
+
+import asyncio
+from asyncio import Future
+
+from textual import events
+from textual.binding import Binding
+from textual.containers import Vertical
+from textual.geometry import clamp
+from textual.message import Message
+from textual.reactive import reactive
+from textual.widgets._list_item import ListItem
+
+
+class ListView(Vertical, can_focus=True, can_focus_children=False):
+    DEFAULT_CSS = """
+    ListView {
+        scrollbar-size-vertical: 1;
+    }
+    """
+
+    BINDINGS = [
+        Binding("down", "down", "Down"),
```

Comment:
In the tree view I've called these `cursor_up` and `cursor_down`. I think we should adopt that as a convention. So we can standardise bindings.

Reply:
I've updated them all to match the tree view exactly.